chalicelib/action.py

```diff
@@ -49,10 +49,12 @@ def perform_action(self):
         """
 
         self.action = self.params[0]
-        log.debug(f"Action is: {self.action}")
+        self.arguments = self.params[1:]
```

Comment:
What happens if params only has a single value? Like ["help"]

Reply:
`self.arguments` Will then be an empty list